workshops/templates/forms/autoupdateprofile.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base_nonav_fixed.html" %}
+
+{% load crispy_forms_tags %}
+{% load selectable_tags %}
+
+{% block content %}
+{% crispy form form_helper %}
+{% endblock %}
+
+{% block extrajs %}
+{{ form.media }}
+{% endblock extrajs %}
```

Comment:
We have, but it inherits from `base_nav_fixed.html`. I've renamed this template to `generic_form_nonav.html`.